controllers/app/api/v1/appaction_types.go

```diff
@@ -23,48 +23,42 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
-// ActionsSpec defines the desired state of Actions
-type ActionsSpec struct {
+// AppActionSpec defines the desired state of AppAction
+type AppActionSpec struct {
```

Comment:
I think the name `Action` is better then `AppAction` because Action maybe not only for app~

Reply:
ok